examples/asr/conf/speech_to_text_hf_finetune.yaml

```diff
@@ -0,0 +1,175 @@
+  rnnt_reduction: 'mean_volume'
+  skip_nan_grad: false
+
+  # configs for huggingface load_dataset function
+  data_path: "librispeech_asr"
+  data_name: null
+  num_proc: 8  # Number of processes when downloading and generating the dataset locally.
+  streaming: false # set True to use streaming mode, which doesn't wait for data downloading but each training step takes longer in the first epoch. If True, you'll need to specify trainer.max_steps instead of trainer.max_epochs.
+
+  # keys for audio, sample_rate and text in the huggingface dataset, keys seperated by `.` for nested fields.
+  audio_key: "audio.array"
+  sample_rate_key: "audio.sampling_rate"
+  text_key: "text"
+
+  # simple text cleaning, by default converts all chars to lower-case and only keeps alpha-numeric chars.
+  normalize_text: true
+  symbols_to_keep: "'"
+
+  train_ds:
+    normalize_text: ${model.normalize_text}
+    symbols_to_keep: ${model.symbols_to_keep}
+    manifest_filepath: "hugginface"  # set to a not None value to avoid breaking existing code
+    audio_key: ${model.audio_key}
+    sample_rate_key: ${model.sample_rate_key}
+    text_key:  ${model.text_key}
+    hf_data_cfg:  # configs for huggingface load_dataset, add more if needed
+      path: ${model.data_path}
+      name: ${model.data_name}
+      split: 'validation.clean'
+      streaming: ${model.streaming}
+      num_proc: ${model.num_proc}
+  
+    sample_rate: ${model.sample_rate}
+    batch_size: 16 # you may increase batch_size if your memory allows
+    shuffle: true
+    num_workers: 0
+    pin_memory: true
+    use_start_end_token: false
+
+  validation_ds:
+    normalize_text: ${model.normalize_text}
+    symbols_to_keep: ${model.symbols_to_keep}
+    manifest_filepath: "hugginface"  # set to a not None value to avoid breaking existing code
+    audio_key: ${model.audio_key}
+    sample_rate_key: ${model.sample_rate_key}
+    text_key:  ${model.text_key}
+    hf_data_cfg:  # configs for huggingface load_dataset, add more if needed
+      path: ${model.data_path}
+      name: ${model.data_name}
+      split: 'validation.clean'
+      streaming: ${model.streaming}
+      num_proc: ${model.num_proc}
+
+    sample_rate: ${model.sample_rate}
+    batch_size: 16
+    shuffle: false
+    num_workers: 0
+    pin_memory: true
+    use_start_end_token: false
+
+  test_ds:
+    normalize_text: ${model.normalize_text}
+    symbols_to_keep: ${model.symbols_to_keep}
+    manifest_filepath: "hugginface"  # set to a not None value to avoid breaking existing code
+    audio_key: ${model.audio_key}
+    sample_rate_key: ${model.sample_rate_key}
+    text_key:  ${model.text_key}
+    hf_data_cfg:  # configs for huggingface load_dataset
+      path: ${model.data_path}
+      name: ${model.data_name}
+      split: 'test.other'
+      streaming: ${model.streaming}
+      num_proc: ${model.num_proc}
+
+    sample_rate: ${model.sample_rate}
+    batch_size: 16
+    shuffle: false
```

Comment:
Val and resr batch sizes should be 8 or 4

Reply:
done